### DISCUSSION TITLE
PnP Search v4 query to exclude Meeting Rooms from the search results

### DISCUSSION THREAD

**Discussion by slawulo** [microsoft-search/pnp-modern-search#2980] [Q&A]
This might be a silly question, but how can i modify my search query to exclude meeting rooms from the result in PnP People Search?

The current query is as follow:

{searchTerms} (WorkEmail:"*domainA.com" OR WorkEmail:"*domainB.com"  OR WorkEmail:"*domainC.com")

It is working as desired, besides that room and meeting rooms object are being displayed as well

**Comment by kasperbolarsen** [comment]
Not a silly question at all. In order to exclude accounts you must first know your data. I suggest using either the SharePoint Search Query tool or the chrome extension SP Editor. 
Once you know your data you might find some inspiration at https://docs.intraactive.net/?doc=/intraactive-intranet/phonebook-results/

Or similar sources

**Comment by StephenHoughton4** [comment]
It depends how your data is set for the meeting rooms. For example in my circumstance I didnt want Admin Accounts, so did the following:

{SearchTerms}* -AccountName:Admin*

Which removed any accounts starting with Admin...
